package.json

```diff
@@ -4,10 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
@a-braham to avoid redundancy lets have this command as `"jest --coverage"` and do away with line 11

Reply:
we would then have scripts in `.travis.yml` as only `npm run test`